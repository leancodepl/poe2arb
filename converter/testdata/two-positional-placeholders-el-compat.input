[
    {
        "term": "textWithPositionalPlaceholders",
        "definition": "So {} is a {}.",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    }
]