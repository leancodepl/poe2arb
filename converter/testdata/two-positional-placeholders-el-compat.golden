{
    "@@locale": "en",
    "textWithPositionalPlaceholders": "So {pos0} is a {pos1}.",
    "@textWithPositionalPlaceholders": {
        "placeholders": {
            "pos0": {
                "type": "Object"
            },
            "pos1": {
                "type": "Object"
            }
        }
    }
}
